spec/rails_helper.rb

```diff
@@ -48,6 +48,7 @@
   "selenium-release.storage.googleapis.com",
   "developer.microsoft.com/en-us/microsoft-edge/tools/webdriver",
   "api.knapsackpro.com",
+  "elasticsearch:9200",
```

Comment:
We are in the process of [setting up Docker for our CI](https://github.com/forem/forem/compare/mstruve/buildkite-ci#diff-053aa3ea46cb7c6a649e0d0fc592ce729b66e1d44be475516fd58857ba695862R51) and this is one of the things I also had to add so +1 for doing it here. I think you might also need to add redis here as well. Worth checking.

Also, I believe you only need the host, `elasticsearch` and not the port.

Reply:
> Also, I believe you only need the host, `elasticsearch` and not the port.

True, I added the port to be more specific but in this case it shouldn't be an issue.

> I think you might also need to add redis here as well. Worth checking.

It's possible I missed it because I get some other unrelated failures such as missing Chrome binary. However I ran the entire RSpec suite with my Docker dev setup and I didn't see a VCR or WebMock failure for Redis requests, so I'm not sure it's needed. At least not until someone explicitly makes some Redis requests from RSpec.